src/helpers/index.tsx

```diff
@@ -64,17 +64,15 @@ export async function getV1MarketPrice() {
  * @returns INTEGER usd value
  */
 export async function getTokenPrice(tokenId = "olympus") {
-  let resp;
-  try {
-    resp = await axios.get(`https://api.coingecko.com/api/v3/simple/price?ids=${tokenId}&vs_currencies=usd`);
-    return resp.data[tokenId].usd;
-  } catch (e) {
-    // console.log("coingecko api error: ", e);
-  }
+  const resp = (await axios.get(`https://api.coingecko.com/api/v3/simple/price?ids=${tokenId}&vs_currencies=usd`)) as {
+    data: { [id: string]: { usd: number } };
+  };
+  let tokenPrice: number = resp.data[tokenId].usd;
+  return tokenPrice;
 }
 
 export async function getTokenIdByContract(contractAddress: string) {
-  let resp;
+  let resp: any;
   try {
```

Comment:
error handling still present here on this similar func

Reply:
I was getting a TypeScript static check error that `resp` was `unknown` type. When I added `any` to the declaration of `resp` the error went away.

Reply:
shouldn't we avoid using any as a type?

Reply:
for reference: https://stackoverflow.com/questions/62217642/react-and-typescript-which-types-for-axios-response

Reply:
agreed with jem. This type should be `string` for that particular api response https://www.coingecko.com/en/api/documentation

Reply:
> shouldn't we avoid using any as a type?

Good point. See suggested update.